dkg/dkg.go

```diff
@@ -379,7 +384,17 @@ func signAndAggDepositData(ctx context.Context, ex *exchanger, shares []share, w
 		return nil, err
 	}
 
-	aggSigDepositData, err := aggDepositDataSigs(peerSigs)
+	pubkeyToShares := make(map[core.PubKey]map[int]*bls_sig.PublicKey)
```

Comment:
would pass in shares into `aggDepositDataSigs` and create this map there since this function is getting too big

Reply:
done